### PR TITLE
Add meter component

### PR DESCRIPTION
**Pull request by styczynski**
Add gauge meter using [react-svg-gauge](https://www.npmjs.com/search?q=react-svg-gauge)

image

### CODE REVIEW

File: frontend/src/components/universal/Meter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    render() {
+        
+        const GaugeComponent = Gauge as any;
+        
+        return (
+            <Wrapper>
+                <Tooltip
+                    content={this.props.tooltipContent}
+                >
+                    <div>
```
Comment (mbalc) [inline frontend/src/components/universal/Meter/index.tsx]: redundant div?

File: frontend/src/components/universal/Meter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    tooltipContent?: React.ReactElement<any>;
+};
+
+class Meter extends React.Component<MeterProps, undefined> {
+    render() {
+        
+        const GaugeComponent = Gauge as any;
+        
+        return (
+            <Wrapper>
```
Comment (mbalc) [inline frontend/src/components/universal/Meter/index.tsx]: padding: 0; seems hacky here

File: frontend/src/components/universal/Meter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+const Wrapper = styled.div`
+  padding: 0;
+`;
+
+export interface MeterProps {
+    theme: any;
+    label: string;
+    value: number;
+    format?: (number) => string;
+    tooltipContent?: React.ReactElement<any>;
```
Comment (mbalc) [inline frontend/src/components/universal/Meter/index.tsx]: let's check if we allow pure strings this way here

File: frontend/src/components/universal/Meter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        
+        return (
+            <Wrapper>
+                <Tooltip
+                    content={this.props.tooltipContent}
+                >
+                    <div>
+                        <GaugeComponent
+                            value={this.props.value}
+                            width={200}
```
Comment (mbalc) [inline frontend/src/components/universal/Meter/index.tsx]: magic variables?

File: frontend/src/components/universal/index.tsx [hunk 5]
```diff
@@ -37,5 +38,6 @@ export default {
     Breadcrumbs,
     SearchableList,
     Card,
-    Tooltip
+    Tooltip,
+    Meter
```
Comment (mbalc) [inline frontend/src/components/universal/index.tsx]: how about using dangling commas?

Comment (mbalc) [inline frontend/src/components/universal/index.tsx]: maybe even eslint this

File: frontend/src/components/universal/Meter/index.tsx (hunk 2 above)
Comment (styczynski) [reply frontend/src/components/universal/Meter/index.tsx]: This was redundant. Fixed.

File: frontend/src/components/universal/Meter/index.tsx (hunk 1 above)
Comment (styczynski) [reply frontend/src/components/universal/Meter/index.tsx]: Removed dead code div.

File: frontend/src/components/universal/Meter/index.tsx (hunk 4 above)
Comment (styczynski) [reply frontend/src/components/universal/Meter/index.tsx]: Moved this to uppercased-constants in the same file and set as defaults (for props override)

File: frontend/src/components/universal/index.tsx (hunk 5 above)
Comment (styczynski) [reply frontend/src/components/universal/index.tsx]: This is issue connected with #56

Comment (styczynski) [reply frontend/src/components/universal/index.tsx]: For now, it won't be changed until linter will be introduced. See #56.

File: frontend/src/components/universal/Meter/index.tsx (hunk 3 above)
Comment (styczynski) [reply frontend/src/components/universal/Meter/index.tsx]: Yeah pure strings are fine with that.

**Pull request merged**